modules/ROOT/pages/cache-scope.adoc

```diff
@@ -41,7 +43,9 @@ Note that if the generated response is a nonrepeatable stream, the scope does no
 
 ==  Caching Strategy
 
-A Caching Strategy defines the actions a Cache scope takes when a message enters its sub-flow. By default, the Cache scope uses a Caching Strategy that stores data in an InMemoryObjectStore instead of an ObjectStore. MuleSoft recommends that you only use the default for nonproduction purposes, such as testing. For production environments, you should set up a Caching Strategy that uses the ObjectStore.
+A Caching Strategy defines the actions a Cache scope takes when a message enters its sub-flow. By default, the Cache scope uses a Caching Strategy that stores data in an InMemoryObjectStore instead of an ObjectStore. For production environments, set up a Caching Strategy that uses an ObjectStore. Use the default strategy only for testing purposes.
```

Comment:
InMemoryObjectStore and ObjectStore should be in monospaced characters. Also, if these are attributes name, should not have a determiner:
```
the Cache scope uses a Caching Strategy that stores data in an in memory object store (`InMemoryObjectStore`) instead of a regular object store (`ObjectStore`).
```
Or something like that.